src/v0/destinations/braze/config.js

```diff
@@ -18,7 +18,7 @@ function getTrackEndPoint(endPoint) {
 }
 
 function getSubscriptionGroupEndPoint(endPoint) {
-  return `${endPoint}/subscription/status/set`;
+  return `${endPoint}/v2/subscription/status/set`;
```

Comment:
Is this api compatible with the previous one?

Reply:
yes, updated transformer code to work with previous payload from user and transforming that code as per this api